src/createServer.js

```diff
@@ -1,11 +1,37 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { userRouter } = require('./routes/users.route');
+const { expensesRouter } = require('./routes/expenses.route');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  /* CORS  */
+  const options = {
+    origin: '*',
+    methods: 'GET, POST, PUT, DELETE , PATCH',
```

Comment:
The `methods` option in the CORS configuration contains a space before the 'PATCH' method. Consider removing the extra space to ensure consistency.